manim/_config/utils.py

```diff
@@ -312,6 +559,30 @@ def digest_parser(self, parser):
         return self
 
     def digest_args(self, args):
+        """Process the config options present in CLI arguments.
+
+        Parameters
+        ----------
+        args : :class:`argparse.Namespace`
+            An object returned by :func:`.main_utils.parse_args()`.
+
+        Returns
+        -------
+        self : :class:`ManimConfig`
+            This object, after processing the contents of ``parser``.
+
+        See Also
+        --------
+        :func:`.main_utils.parse_args()`, :meth:`~ManimConfig.digest_parser`,
+        :meth:`~ManimConfig.digest_file`
+
+        Notes
+        -----
+        If ``args.config_file`` is non-null, ManimConfig tries to digest the
```

Comment:
non-null does not sound very pythonic. "not ``None``"? Or maybe even "If ``args.config_file`` is set, ...?

Reply:
```suggestion
        If ``args.config_file`` is a non-empty string, ``ManimConfig`` tries to digest the
```